src/api/spec/factories/bs_requests.rb

```diff
@@ -166,6 +166,40 @@
 
     factory :bs_request_with_maintenance_incident_action do
```

Comment:
Taking the idea from @eduardoj's PR, I'd use plural in the factory name.

```suggestion
    factory :bs_request_with_maintenance_incident_actions do
```

Reply:
Fixed